netbox/templates/extras/object_journal.html

```diff
@@ -6,7 +6,7 @@
 
 {% block content %}
   <div class="card">
-    <div class="card-body table-responsive">
+    <div class="card-body table-responsive journal-table">
```

Comment:
Big :-1: on bespoke CSS classes. Instead, can we either a) restore the margin universally, or b) introduce a separate utility class that removes the margin where needed?